entities/tax.rb

```diff
@@ -0,0 +1,19 @@
+module Entities
+  class Tax
+    USUAL_TYPE = 'usual'.freeze
+    CAPITALIST_TYPE = 'capitalist'.freeze
+    VIRTUAL_TYPE = 'virtual'.freeze
+
+    def initialize(type)
+      @type = type
+    end
+
+    def tax(_amount)
+      raise NotImplementedError 'Method tax not implemented'
+    end
+
+    def default_tax
+      0
```

Comment:
It would be better to use constant for '0' value. If you need it elsewhere within application, you don't need to initialize new instance of the class and get it, you could only use this constant.